qutebrowser/javascript/history.js

```diff
@@ -23,8 +23,12 @@ window.loadHistory = (function() {
     // Date of last seen item.
```

Comment:
@imransobir Want to look at those changes again? I kind of lost track here, but it looks like `qute://history` works fine?